src/js/components/header/PageTabs.jsx

```diff
@@ -0,0 +1,19 @@
+// @flow
```

Comment:
Not sure what we could do that would increase the clarity compared to storybooks alone, but I'm definitely open to suggestions. Ideally I'd like to have a lot more flexibility in JSX so we could set properties and/or have multiple different sets of children, something that's really nice in MXML. You could have:
```
<Header>
  <titleBar>....</titleBar>
  <pageTabs>....</pageTabs>
  <detailView>....</detailView>
</Header>
```
And so on and so forth, with all the lowercase elements being properties of Header, and within its namespace, rather than being actual standalone components. Being forced to only have a single root element returned from `render()` is another thing complicating it IMO.